4.0/en/0x10-V1-Architecture.md

```diff
@@ -50,7 +50,9 @@ This is a placeholder for future architectural requirements.
 
 ## V1.5 Input and Output Architectural Requirements
 
-In 4.0, we have moved away from the term "server-side" as a loaded trust boundary term. The trust boundary is still concerning - making decisions on untrusted browsers or client devices is bypassable. However, in mainstream architectural deployments today, the trust enforcement point has dramatically changed. Therefore, where the term "trusted service layer" is used in the ASVS, we mean any trusted enforcement point, regardless of location, such as a microservice, serverless API, server-side, a trusted API on a client device that has secure boot, partner or external APIs, and so on. 
+In 4.0, we have moved away from the term "server-side" as a loaded trust boundary term. The trust boundary is still concerning - making decisions on untrusted browsers or client devices is bypassable. However, in mainstream architectural deployments today, the trust enforcement point has dramatically changed. Therefore, where the term "trusted service layer" is used in the ASVS, we mean any trusted enforcement point, regardless of location, such as a microservice, serverless API, server-side, a trusted API on a client device that has secure boot, partner or external APIs, and so on.
+
+The "untrusted client" term here refers to client-side technologies that render the presentation layer, commonly refered to as 'front-end' technologies. The term "serialization" here not only refesr to sending data over the wire like an array of values or taking and reading a JSON structure, but also passing complex objects which can contain logic. 
```

Comment:
@csfreak92  I have trimmed this paragraph a bit. Do you think it still makes sense?

Reply:
@tghosth, all seems good. There's just a typo on "refers" on the very last sentence, otherwise this is good. Are we still ok to push this commit for the explanatory text for this V1.5 Input and Output Architectural Requirements section if it was proposed by Jim to remove 1.5.2?

Reply:
Thanks for noticing the typo. I think we can push this as I am proposing to leave 1.5.2 in but maybe I will leave it open a little longer in case there is any pushback.